package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -58,30 +58,6 @@ dhcp_calc() {
 	echo "$res"
 }
 
-dhcp_check() {
-	local ifname="$1"
-	local stamp="${BASEDHCPSTAMPFILE_CFG}.${ifname}.dhcp"
-	local rv=0
-
-	[ -s "$stamp" ] && return $(cat "$stamp")
-
-	# If there's no carrier yet, skip this interface.
-	# The init script will be called again once the link is up
-	case "$(devstatus "$ifname" | jsonfilter -e @.carrier)" in
-		false) return 1;;
-	esac
-
-	udhcpc -n -q -s /bin/true -t 1 -i "$ifname" >&- && rv=1 || rv=0
-
-	[ $rv -eq 1 ] && \
-		logger -t dnsmasq \
-			"found already running DHCP-server on interface '$ifname'" \
-			"refusing to start, use 'option force 1' to override"
-
-	echo $rv > "$stamp"
-	return $rv
-}
-
```

Comment:
The makes `force` more than the default, but the only option right? I thought this check is to prevent conflicts that can cause real grief in connecting to our device, debugging, and fixing what are often each our own stupid mistakes. (1) something really messed up in configuration. (2) plugging your WAN cord into a LAN port (or other routed chain).